### PR TITLE
Topic Issues

### PR DESCRIPTION
**Pull request by senaarth**
This pull request is willing to solve the issues about the user stories from sprint 3 (#97, #98, #99, #100, #101, #102, #103, #104) that were about: 

- [x] Create Topic
- [x] Edit Topic
- [x] Delete Topic
- [x] Upvote and Downvote in topic

- [x] Create comment in a topic
- [x] Edit comment made
- [x] Delete comment made
- [x] Upvote and Downvote in comment

### CODE REVIEW

**Comment by senaarth** [comment]
Features already finished and branch with no conflicts with develop, please review this PR.

File: src/routes/itemRoutes.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const { Router } = require('express');
 const router = Router();
-const Item = require('../models/item');
+const Item = require('../models/Item');
```
Comment (guilhermedelyra) [inline src/routes/itemRoutes.js]: o problema é que na models o arquivo 'item' está salvo em minusculo (e nao com o I maiusculo), então dá erro de import

falando nisso, nao seria interessante remover logo todos os scripts referenciando 'Item'?

**Review by guilhermedelyra — CHANGES_REQUESTED** [review]

File: src/routes/itemRoutes.js (hunk 1 above)
Comment (senaarth) [reply src/routes/itemRoutes.js]: Deixei estes arquivos por motivo de nao terem sido feitos por mim, mas pra falar a verdade tambem acho que seria interessante remover logo

**Review by guilhermedelyra — APPROVED** [review]
tudo ok!

**Pull request merged**